libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -1827,33 +1827,34 @@ void ExpressionCompiler::appendExternalFunctionCall(
 	auto funKind = _functionType.kind();
 
 	solAssert(funKind != FunctionType::Kind::BareStaticCall || m_context.evmVersion().hasStaticCall(), "");
-
-	bool returnSuccessCondition = funKind == FunctionType::Kind::BareCall || funKind == FunctionType::Kind::BareCallCode || funKind == FunctionType::Kind::BareDelegateCall || funKind == FunctionType::Kind::BareStaticCall;
+	
```

Comment:
You just removed them ;)

Reply:
Hm, by the way, do you want to create a circleci job that runs your script and checks if it found anything?

Reply:
Yes, that might be nice and such a script should run quickly - I'll look into it!